.github/workflows/preview.yml

```diff
@@ -0,0 +1,60 @@
+name: Update PDF Preview
+
+env:
+  doc_name:
```

Comment:
Same here: missing the ObjVisSAP value for doc_name.

Reply:
Done